config.yml

```diff
@@ -48,20 +48,12 @@ layers:
     sources: [baumbestand_1_cache]
     title: Baumbestand Berlin - Anlagenbäume
 
-  - name: baumbestand_3
-    sources: [baumbestand_3_cache]
-    title: Baumbestand Berlin - Anlagenbäume ohne Datenbankeintrag
-
-  - name: baumbestand_4
-    sources: [baumbestand_4_cache]
-    title: Baumbestand Berlin - Straßenbäume ohne Datenbankeintrag
-
-  - name: baumbestand_0_1_3_4_merged
-    sources: [baumbestand_0_1_3_4_merged_cache]
+  - name: baumbestand_0_1_merged
+    sources: [baumbestand_0_1_merged_cache]
     title: Baumbestand Berlin - Straßenbäume und Anlagenbäume mit und ohne Datenbankeintrag
 
-  - name: baumbestand_0_1_3_4_alkis
-    sources: [baumbestand_0_1_3_4_alkis_cache]
+  - name: baumbestand_0_1_alkis
```

Comment:
actually, i think we should not change this because it will change the URL which is already use in some places…

Reply:
will test this and maybe revert the url part